data/lib/src/extensions/document_extension.dart

```diff
@@ -31,12 +31,13 @@
  *  the Additional Terms applicable to LinShare software.
  */
 
+import 'package:data/src/network/model/local/document_dto.dart';
 import 'package:domain/domain.dart';
 
-extension DocumentOfflineExtensions on DocumentOffline {
+extension DocumentExtensions on Document {
 
-  Document toDocumentFromOffline() {
-    return Document(
+  DocumentDto toDocumentDto() {
```

Comment:
can not understand

Reply:
Convert document to DocumentDto to save it in database